src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -118,5 +118,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     conversationStore.addConversation(conversation);
     response.sendRedirect("/chat/" + conversationTitle);
+
+    String convTest = request.getParameter("Following");
+    System.out.println("User entered conv name: " + conversationTitle);
+    if (convTest == null) {
+      System.out.println("it's null rn");
+    } else if (convTest.equals("Following")) {
+      System.out.print("Following rn");
```

Comment:
Might need a println, else the message won't show up (or be garbled)

Reply:
Still, convTest stays null even after I click the button